### PR TITLE
#164576233 - Setup Dashboard page

### PR DESCRIPTION
**Pull request by jsamchineme**
###  What this PR does
- Allow users access a dashboard after login
- Implement articles listing on dashboard
- write tests for the dashboard component

### How to test
- Clone this branch
- signup and/or login
- From sidebar on dashboard, click top reads to view article listing

### Screenshots

image

### Relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/164576233

### CODE REVIEW

**Review by omoefe-dukuye — COMMENTED** [review]
@jsamchineme Great job man. Loving the dark theme. Though it seems the logic in the `componentDidMount()` hook that exists in `Dashboard.jsx` on staging seems to be missing from your work.

File: src/components/comments/CommentEntries.jsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import PropTypes from 'prop-types';
 import './style.scss';
-import LikeNumberGroup from '../../components/LikeNumberGroup';
-import HorizontalLine from '../../components/HorizontalLine';
+import { LikeNumberGroup } from '../../components/like-button';
```
Comment (ifiokeyo) [inline src/components/comments/CommentEntries.jsx]: @jsamchineme Javascript adopts the camelcase pattern for filenames, directory names, variables, etc. Please change `lines 4 and 5` to use this pattern.

File: src/components/dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 import { connect } from 'react-redux';
+import { SideBar } from './side-bar';
 import PropTypes from 'prop-types';
-import { getOwnProfile } from '../../redux/actions/profile';
 import { toast } from 'react-toastify';
+import { ContentArea } from './content-area';
+import { Switch, Route } from 'react-router-dom';
+import { setLoggedInState } from '../../redux/actions/auth';
+import { getOwnProfile } from '../../redux/actions/profile';
+import { TopReads } from '../articles/top-reads';
```
Comment (ifiokeyo) [inline src/components/dashboard/Dashboard.jsx]: @jsamchineme see above comment.

File: src/components/dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -34,7 +40,16 @@ export class Dashboard extends Component {
   }
 
   render() {
-    return <p>This is the dashboard page placeholder</p>
+    return (
+      <div className={`dashboard-page-wrapper`}>
```
Comment (ifiokeyo) [inline src/components/dashboard/Dashboard.jsx]: @jsamchineme why is the `className` reference this way ?

**Review by ifiokeyo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**